src/sdk/contracts/SimpleStakingContract.ts

```diff
@@ -21,7 +21,7 @@ export function simpleStakingContract(web3: Web3, address: string) {
  * @returns {Promise<string[]>}
  */
 export async function getSimpleStakingContractAddresses(web3: Web3): Promise<string[]> {
-    const chainId = await getChainId(web3)
+    const chainId = await getChainId(web3) // doesn't return correct chainId
```

Comment:
what?

Reply:
getChainId for some reason always returns chainID 1
which doesn't make sense if you have switched network
it doesn't make sense in any way, as the underlying function is nothing more then web3.eth.getChainId

I don't know the reason... yet